messages/ru/yii-validator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
Maybe structure it somehow to have a better idea where a message coming from? For example, add comments / separators:

```php
// AtLeast rule
'Message' => 'Translated message',
````

Reply:
We can make this later.